OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -152,5 +152,16 @@ public override IEnumerable<Target> GetTargets(Actor self)
 		{
 			yield return Target.FromCell(self.World, self.Location);
 		}
+
+		private CPos GetFromSearchLocation(Actor self)
```

Comment:
No explicit `private` keyword, please (that is the default already). Can we also rename this to `GetSearchFromLocation`? It makes more sense imho and the local variable is also called `searchFromLoc`.